src/main/java/org/intellij/sdk/codesync/codeSyncSetup/CodeSyncSetup.java

```diff
@@ -269,6 +272,8 @@ public static boolean checkUserAccess(Project project, String repoPath, String r
                 return validateAccessToken(accessToken);
             } catch (InvalidAccessTokenError invalidAccessTokenError) {
                 // no action needed, user will be prompted for authenticated later by default.
+                PluginState pluginState = StateUtils.getState(repoPath);
```

Comment:
Same comment again, 
I think we discussed to check output of `validateAccessToken(accessToken)` e.g. if it returns false, it means there is no active user. No exception will be raised in this case, kindly correct me @syedgulahmed